benchmark/sql_regressions/DoltRegressionsJenkinsfile

```diff
@@ -0,0 +1,53 @@
+pipeline {
+    agent none
+    stages {
+        stage('Test Dolt nightly') {
+            parallel {
+                stage ("Update Liquidata/dolt-sql-performance:nightly") {
+                    agent {
+                        kubernetes {
+                            label "liquidata-inc-ld-build"
+                        }
+                    }
+                    environment {
+                        PATH = "${pwd()}/.ci_bin:${pwd()}/.ci_bin/node_modules/.bin:${env.PATH}"
```

Comment:
I might leave `${pwd()}/.ci_bin` off here and have the script manage it...see below.